codalab/client/local_bundle_client.py

```diff
@@ -804,15 +802,67 @@ def new_worksheet(self, name, ensure_exists):
             'title': None,
             'frozen': None,
             'items': [],
-            'owner_id': self._current_user_id()
+            'owner_id': '0' if spec_util.is_dashboard(name) else self._current_user_id()
         })
         self.model.new_worksheet(worksheet)
 
         # Make worksheet publicly readable by default
         self.set_worksheet_perm(worksheet.uuid, self.model.public_group_uuid, 'read')
-
+        if spec_util.is_dashboard(name):
+            self.populate_dashboard(worksheet)
         return worksheet.uuid
 
+    def populate_dashboard(self, worksheet):
+        items = [(None, None, u'Welcome to your **CodaLab Dashboard**, which shows worksheets and bundles (programs and datasets) owned by you.  Read the [tutorial](https://github.com/codalab/codalab-worksheets/wiki/User_CodaLab-Worksheets-Tutorial) to learn more.', u'markup'),
```

Comment:
Let's put this in a dashboard.ws file and use worksheet_util.parse_worksheet_form rather than hard coding it here.